### DISCUSSION TITLE
Liquid Base support

### DISCUSSION THREAD

**Discussion by tackITEasy** [ArcadeData/arcadedb#446] [General]
Hello,

Is it possible to use liquidbase with arcadedb ?
Any example?

Thanks a lot

**Comment by lvca** [comment]
I'm sorry but there is no plugin for Liquidbase AFAIK. Any volunteers?

**Reply by clincks** [reply]
I found this : https://github.com/unbroken-dome/liquibase-orientdb

Maybe a good start to make a fork ?

**Reply by lvca** [reply]
Yes, it should be >90% compatible with ArcadeDB if they're using JDBC -> ArcadeDB SQL that was forked from OrientDB. The only changes could be changing the name from "class" to "type".
